src/explore-education-statistics-admin/src/pages/methodology/createMethodologyPage.tsx

```diff
@@ -0,0 +1,223 @@
+import Link from '@admin/components/Link';
+import Page from '@admin/components/Page';
+import dashboardRoutes from '@admin/routes/dashboard/routes';
+import { ContactDetails, IdTitlePair } from '@admin/services/common/types';
+import service from '@admin/services/edit-publication/service';
+import methodologyService from '@admin/services/methodology/service';
+import Button from '@common/components/Button';
+import { FormFieldset, Formik } from '@common/components/form';
+import FormFieldDayMonthYear from '@common/components/form/FormFieldDayMonthYear';
+import Form from '@common/components/form/Form';
+import FormFieldSelect from '@common/components/form/FormFieldSelect';
+import FormFieldTextInput from '@common/components/form/FormFieldTextInput';
+import handleServerSideValidation, {
+  errorCodeToFieldError,
+} from '@common/components/form/util/serverValidationHandler';
+import RelatedInformation from '@common/components/RelatedInformation';
+import SummaryList from '@common/components/SummaryList';
+import SummaryListItem from '@common/components/SummaryListItem';
+import Yup from '@common/lib/validation/yup';
+import { FormikProps } from 'formik';
+import orderBy from 'lodash/orderBy';
+import React, { useEffect, useState } from 'react';
+import { RouteComponentProps } from 'react-router';
+import { CreateMethodologyRequest } from '@admin/services/methodology/types';
+import {
+  DayMonthYearInputs,
+  dateToDayMonthYear,
+  dayMonthYearValuesToInputs,
+  dayMonthYearInputsToDate,
+} from '@common/services/publicationService';
+import { validateMandatoryDayMonthYearField } from '@admin/validation/validation';
+
+interface MatchProps {
+  topicId: string;
+}
+
+interface FormValues {
+  methodologyTitle: string;
+  selectedContactId: string;
+  scheduledPublishDate: DayMonthYearInputs;
+}
+
+const serverSideValidationHandler = handleServerSideValidation(
+  errorCodeToFieldError(
+    'SLUG_NOT_UNIQUE',
+    'methodologyTitle',
+    'Choose a unique title',
+  ),
+);
+
+interface CreateMethodologyModel {
+  contacts: ContactDetails[];
+}
+
+const CreateMethodologyPage = ({
+  match,
+  history,
+}: RouteComponentProps<MatchProps>) => {
+  const [model, setModel] = useState<CreateMethodologyModel>();
+
+  useEffect(() => {
+    Promise.all([service.getPublicationAndReleaseContacts()]).then(
```

Comment:
Only one promise occurring here, so don't need the Promise.all().

Can just write:

`service.getPublicationAndReleaseContacts().then(contacts => `